src/main/java/teammates/common/datatransfer/questions/FeedbackTextQuestionDetails.java

```diff
@@ -13,19 +13,27 @@
     @Nullable
     private Integer recommendedLength;
 
+    private boolean allowRichText;
```

Comment:
boolean variable can start with `is,should,has,can`

```suggestion
    private boolean shouldAllowRichText;
```

Reply:
I think the more important change needed here is `boolean` -> `Boolean`. @xpdavid you should know why.

Reply:
@xpdavid 
Understand. Will just rename all.

@wkurniawan07 
Why is there a need to use Boxed type? Is it coz of the POJO requirement? Will change it regardless.

Reply:
Actual question detail is stored as serialized JSON string in DB. Putting as `boolean` should be fine as when the string is serialized back, the default constructor is called. So there will be no NPE here.

Reply:
So it is fine boxed or unboxed? I tend to leave it boxed for consistency then... thanks for the info.

Reply:
Let do it Boxed to explicitly say the field is `null` for legacy data.

Reply:
> Let do it Boxed to explicitly say the field is null for legacy data.

Wait so should I tag it Nullable? Hmm then how would the legacy data turn out to be?

Anywhere else should I turn it into Boxed type? Or just at data layer?